src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -1,13 +1,11 @@
 package seedu.address.testutil;
 
+import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
I think we should stick with the earlier convention of importing individually or else we will fail the checktyle,